docs/technical-documentation/Security_Assessment.md

```diff
@@ -0,0 +1,89 @@
+# Security Assessment Policy-Hub 
+
+|                           |                                                                                                |
+| ------------------------- | ---------------------------------------------------------------------------------------------- |
+| Contact for product       | [@evegufy](https://github.com/evegufy) <br> [@jjeroch](https://github.com/jjeroch)             |
+| Security responsible      | [Szymon Kowalczyk](szymon.kowalczyk@zf.com) |
+| Version number of product | 0.1.0                                                                                          |
+| Dates of assessment       | 2024-02-16: Assessment                                                                      |
+| Status of assessment      | ASSESSMENT DRAFT                                                                            |
```

Comment:
Look good to me, could you please change the status still to finalized?

Reply:
Done